### PR TITLE
ENDOC-547 7.1 Update v1 and Create new v5: Update the Prj Data Model tutorial

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**
getting concepts/language in place pending finalized jhipster tutorial

### CODE REVIEW

File: vuepress/docs/next/tutorials/create/ms/update-data-model.md [hunk 1]
```diff
@@ -9,22 +9,22 @@ sidebarDepth: 2
 This tutorial explains how to use the [Entando Component Generator](../../../docs/create/component-gen-overview.md) powered by [JHipster](https://www.jhipster.tech/) to quickly update the data model for your Entando project.
 
 ## Prerequisites
-Leverage an existing project or [create a project](./generate-microservices-and-micro-frontends.md) and navigate to the root folder. The steps below assume you're working in the top-level project directory.
+A [Blueprint-generated project](./generate-microservices-and-micro-frontends.md) 
 
 ## Tutorial
-1. Use JHipster to extract the current application description. The resulting JHIpster Domain Language (JDL) file contains your project's application configuration and entity definitions.
+The steps below assume you're working out of the root directory of the microservice, e.g. /microservices/conference-ms.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/create/ms/update-data-model.md:15]: root directory should be just /microservices or maybe it should read root directory of the conference microservice?

File: vuepress/docs/next/tutorials/create/ms/update-data-model.md [hunk 2]
```diff
@@ -45,26 +45,17 @@ relationship OneToMany {
...
-
-6. You can now build your updated project and [run it locally](./run-local.md) or [deploy it to Entando](../pb/publish-project-bundle.md). To execute a comprehensive local test, use the following commands to build the project
-```
-ent prj build
-ent prj xk start
-ent prj be-test-run
-ent prj fe-test-run
-```
-then start Keycloak, the microservice, and one of the MFEs.
+If the default project structure has been retained, this step will update your data model, add entries to Liquibase to upgrade database schema during deployment, add service methods to your microservice, add fields to your MFEs, etc.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/create/ms/update-data-model.md:56]: should there be an 'and' in this kind of list?

Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/create/ms/update-data-model.md:56]: it's a little awkward - almost too detailed and comprehensive to end with an "etc." - but the structure avoids an "and"

File: vuepress/docs/next/tutorials/create/ms/update-data-model.md (hunk 1 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/create/ms/update-data-model.md:15]: actually if the MS is conference-ms then /microservices/conference-ms is the path to the root of the MS; same as your second rec but including the path

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**